src/main/java/co/inventorsoft/StreamHomework.java

```diff
@@ -29,7 +35,12 @@
      * @return collection of user, without duplicates
      */
     public List<User> createUsers(final List<String> emails) {
-        return null;
+        return emails != null ? emails.stream()
+                .distinct()
+                .filter(Objects::nonNull)
```

Comment:
In this particular case it doesn't really matter but in general if you have to apply filter and distinct operations it's always better to do filtering before checking uniqueness since it reduces the amount of elements which should be processed. Did you get the idea?